Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -429,6 +429,21 @@ private function detectIndent(Tokens $tokens, $index)
             if ($prevToken->isGivenKind($goBackTokens)) {
                 return $this->detectIndent($tokens, $prevIndex);
             }
+
+            if ($token->isClassy() && $prevToken->isGivenKind(T_NEW)) {
+                for ($prevIndex = $prevIndex - 1; 0 <= $prevIndex; --$prevIndex) {
+                    $prevToken = $tokens[$prevIndex];
+                    // if token is multiline whitespaces
+                    if ($prevToken->isWhitespace() && !$prevToken->isWhitespace(array('whitespaces' => " \t"))) {
```

Comment:
we do https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Tokenizer/Token.php#L439
^_^